packages/tasks/src/model-libraries.ts

```diff
@@ -179,6 +179,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "doctr",
 		repoUrl: "https://github.com/mindee/doctr",
 	},
+	edge: {
+		prettyLabel: "Edge",
+		repoName: "Edge",
+		repoUrl: "https://github.com/cartesia-ai/edge",
```

Comment:
Is this a "soon to be released" repository?

Reply:
Yes.